### DISCUSSION TITLE
It works! On a Raspberry Pi Model B+ V1.2

### DISCUSSION THREAD

**Discussion by airlomba** [jekhokie/raspberry-noaa-v2#427] [Show and tell]
Hello all,

I have the code working on a Raspberry Pi 1.2 (without overclock)
After each pass, the Pi take about 10~15 minutes to process the information and create all the images. I will try to measure this time with more accuracy.

However, METEOR decoding is not possible on this Pi model due to lack of RAM available... I will try to find some turn around to try to solve this issue; any suggestions are welcome.

You may look at an image from last NOAA 18 pass "over" my shack at: [http://www.airlomba.net/radio/satellite/images/WX_NOAA/NOAA-18-20210811-120250-MCIR.jpg](http://www.airlomba.net/radio/satellite/images/WX_NOAA/NOAA-18-20210811-120250-MCIR.jpg)

Many thanks to all contributors for your good work.
Best regards.

**Comment by airlomba** [comment]
Hello all,
I have made some accurate measurements on the processing time used by this application (only) on a Raspberry Pi v1.2
So the time of reception depends on the max elevation of the satellite pass. The results are the following:

1) On a **low pass of about 15 degrees** elevation (the station location does not appear on the picture), the full time (reception and post-processing) is about **30 minutes**.
2) On a **high pass** (over 70 degrees elevation), the duration goes **between 70 and 80 minutes**.

I hope this may be interesting for someone... :)
Best regards,

**Comment by ja-Bertolin** [comment]
Hi,

I have a B version too and I was wondering how to install all the package on it. I installed the image but CPU takes 92% without any additional apps, with Gpredict is 100%,.

Which SO are you using?  SO , is it feasible to still make it run on such old RPi?

Thanks!

**Reply by airlomba** [reply]
> Hi,
> 
> I have a B version too and I was wondering how to install all the package on it. I installed the image but CPU takes 92% without any additional apps, with Gpredict is 100%,.
> 
> Which SO are you using? SO , is it feasible to still make it run on such old RPi?
> 
> Thanks!

Hi,
This application is very CPU demanding. So, It is normal for the CPU to be at 100%.

My OS is Raspbian (buster), working headless (no monitor) and communicating via SSH.
It was for me an interesting try-out but I do not recommend it if you want to get all passes, because you need a lot of CPU to process images between passes and, as stated in my previous posts, it take a long time (up to 30 minutes). So forget about one of two consecutive passes (in this case, I sugest you choose the one with highest elevation).

Because I also wanted to get METEOR images, I am now using a Raspberry Pi 4 (2GB of RAM), headless, with the lastest RaspiOS (also Debian based). However, since last storm a couple of weeks ago, my reception is bad and I need to build a new antenna...

I hope this message is somewhat useful to you. Anyway, I'll be glad to answer eventual questions you may have yet.

Happy New Year! :)
Best, regards.

**Reply by ja-Bertolin** [reply]
Happy New Year Emmanuel and thanks a lot for your comments

Let’s see whether I understood them, 

- You have Buster, so you did not use the image. I understand that you recompile it everything. Do you think that I could do it with previous version such as Jessie? (that is not so high resources demanding) 
- When you mention “communicating via SSH”, do you mean that all the processes are executed via command line? So I do no need to run any graphical app and gpredict/wxtoimage could be run via command line too.

I am aware that due to the scarce resources the time to process every pass is so long that you do not have time to get the closest one. I do not expect to run it un unattended mode, so I will try to choose the best ones.

I was  thinking to buy a new RPi (last one) but I’d rather to give a try to the old one.

Best

**Reply by airlomba** [reply]
Hi,

I can't tell if it compiles on Jessie as it may be lacking some libraries more recent than Jessie last version. Give it a try. :)
About SSH... The Raspberry is just connected to my LAN, without keyboard nor screen. I access to it via SSH to configure and tweak it. The results can bee accessed via a web browser running in any device connected to your LAN (even from Internet, if you have port forwarding configured in your router). So, there is no need to install the graphical interface of Linux in your Pi (unless you need it for something else).
Yes! wxtoimage runs as necessary, invoqued by the scripts from raspberry-NOAA-v2. Also, you don't need access to Gepredict GUI, as the passes scheduling are made in the background. In short, you just need to install the minimal version of Raspbian and then compile the app from source, configure it and you are good to go.

My Pi is working 24/7 standalone. With a Pi 4, I can have other applications running as well.

Regards.

**Comment by ja-Bertolin** [comment]
Hi,

Great, glad to hear that. So I think that I will keep the bullseye installation but with no GUI and I will try to understand how scripts work with raspberry-NOAA-v2 and run just with the minimum. 

Thanks a lot for your help, my old RPi will have again a space in my shack to play with it.

Regards
